app/seeders/basic_data/role_seeder.rb

```diff
@@ -114,6 +110,10 @@ def member
       }
     end
 
+    def project_admin
+     { name: I18n.t(:default_role_project_admin), position: 4, permissions: Role.new.setable_permissions.map(&:name) }
+    end
+
     def reader
```

Comment:
The wp states to have the order:
* member
* reader
* project_admin

This PR currently changes it to:
* member
* project_admin
* reader